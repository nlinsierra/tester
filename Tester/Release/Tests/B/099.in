4S 5D 9C 6S TD KS AD 4C AC JC 6D 6C 8D 9D 3C QC 3S 7S 2D TS 7H TH 6H 3H QH 8C JD AS
