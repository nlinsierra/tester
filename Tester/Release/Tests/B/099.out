24
2C 5C 7C TC KC 3D 4D 7D QD KD 2H 4H 5H 8H 9H JH KH AH 2S 5S 8S 9S JS QS
